packages/parser-typescript-config/tests/tests.ts

```diff
@@ -34,7 +68,7 @@ test(`If the resource doesn't match the regex, nothing should happen`, async (t)
     sandbox.restore();
 });
 
-test('If the file contains an invalid json, it should fail', async (t) => {
+test.serial('If the file contains an invalid json, it should fail', async (t) => {
```

Comment:
Any particular reason these were switched to be `serial`?

Reply:
If you stub the same object in different tests that run in parallel, you get an error because the object is already stubbed.

Reply:
Anyway, I'm removing some unnecessary `.serial`

Reply:
Got it. 

Not that you need to do anything now, but for future note you can work around this by creating the stubbed object on a per-test basis. In this case that would also move the proxyquire and subsequent import call into a helper. You can see [`extension-browser/tests/background-script.ts`](https://github.com/webhintio/hint/blob/e1c8c4037ab45a7403ab33c70b61e265446bd632/packages/extension-browser/tests/background-script.ts#L16) for an example along these lines.